### PR TITLE
fix: we shouldnt rely on anything

### LINKED ISSUE #911: Notebooks: Flux script with without _measurement and _field breaks SimpleTable
any query that doesn't have `_measurement` or `_field` in the response will break SimpleTable.

image

```
from(bucket: "telegraf")
  |> range(start: -1m, stop: now())
  |> filter(fn: (r) => r._measurement == "cpu")
  |> group(columns: ["cpu"])
  |> count()
```
sample data:
```
#group,false,false,true,true,true,false,false
#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,string,double,dateTime:RFC3339
#default,_result,,,,,,
,result,table,_start,_stop,location_name,_value,_time
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T04:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T05:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T06:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T07:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T08:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T09:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T10:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T11:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T12:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,0,2021-03-18T13:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T14:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T15:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T16:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T17:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T18:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,,2021-03-18T19:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,1.1082252799999999,2021-03-18T20:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,3.9826846,2021-03-18T21:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,0,2021-03-18T22:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,3.5671001199999997,2021-03-18T23:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,0.32900438,2021-03-19T00:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,10.09523966,2021-03-19T01:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,30.701303460000005,2021-03-19T02:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,5.43723028,2021-03-19T03:00:00Z
,,0,2021-03-18T03:04:53.552394796Z,2021-03-19T03:04:53.552394796Z,SF Bernal,0,2021-03-19T03:04:53.552394796Z
```

### PR DESCRIPTION
**Pull request by drdelambre**
Closes #911

### CODE REVIEW

File: src/visualization/types/SimpleTable/PagedTable.tsx [hunk 1]
```diff
@@ -118,6 +118,7 @@ const subsetResult = (
         subset['_field'],
         subset['_value'],
       ]
+        .filter(c => !!c)
```
Comment (genehynson) [inline src/visualization/types/SimpleTable/PagedTable.tsx:121]: remove this change and I think the PR looks good 👍 😛

Comment (genehynson) [reply src/visualization/types/SimpleTable/PagedTable.tsx:121]: jkkkkkkkkkk

**Review by genehynson — APPROVED** [review]
🚀

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 37 | :red_circle:
accessibility | 96 | :green_circle:
best-practices | 93 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :question:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?vars%5Blighthouse_score_branch%5D=915)

**Pull request merged**